lib/httprequest.js

```diff
@@ -5,59 +5,66 @@ var axios = require("axios");
 var debugLog = false;
 
 var HttpRequest = function (host, timeout, apiVersion) {
-    this.host = host || "http://localhost:8685";
-    this.timeout = timeout || 0;
-    this.apiVersion = apiVersion || "v1";
+	this.host = host || "http://localhost:8685";
+	this.timeout = timeout || 0;
+	this.apiVersion = apiVersion || "v1";
```

Comment:
should avoid unnecessary changes, such as indent style.

Reply:
Sorry visual studio code set as format on save. İ can fix.